### PR TITLE
Refresh application

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
I want to try out here the 1-4-stable branch in activeadmin before releasing it, but I'm refreshing this app a bit first.

* Removes a bunch of unused gems.
* Removes hound in favor of plain rubocop.
* Removes warnings about factory girl by migrating to factory bot.
* Removes warnings about sqlite stuff by using postgreSQL in all environments.
* Bumps to ruby 2.5.3.
* Bump airbrake-ruby and ignore 404 errors from its reports.

### CODE REVIEW

**Comment by javierjulio** [comment]
Love it! Thanks! 😍 Let me know when this is ready and I'll approve for you.

File: config/database.yml [hunk 1]
```diff
@@ -1,25 +1,17 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
```
Comment (workgena) [inline config/database.yml]: Just tested local. It is easy to setup. Rails raised an error only on missing `host` variable:

```yml
host: localhost
```

Comment (deivid-rodriguez) [reply config/database.yml]: Thank you so much for having a look @workgena, I'll include this missing variable!

Comment (deivid-rodriguez) [reply config/database.yml]: Updated in 0739d65f9717f0d5f736e661fd02d980a8c07872!

**Comment by deivid-rodriguez** [comment]
@javierjulio This is ready! :)

**Comment by deivid-rodriguez** [comment]
Great, I'll try to deploy it now!

**Comment by deivid-rodriguez** [comment]
I deployed the new version and I can't see any obvious regression. As a next step for this repo, I'd like to set up autodeploy to heroku, but I'll go back to releasing activeadmin 1.4 now. :)

**Comment by javierjulio** [comment]
@deivid-rodriguez that sounds like a great idea! 👍🏻 Auto deploy would be really useful. We could also have Heroku create a test environment for when you submit a PR here. That would be cool.

**Pull request merged**